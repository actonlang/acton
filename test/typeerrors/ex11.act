def f(s):
    return s + "!"

a = f([8])
