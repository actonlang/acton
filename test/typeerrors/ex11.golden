Building file ../test/typeerrors/ex11.act
  Compiling ex11.act for release

ERROR: Error when compiling ex11 module: Type error

  |
1 |def f(s):
  |    ^


  |
4 |a = f([8])
  |    ^^^^^^
Type incompatibility between definition of and call of f (__builtin__.list[_] must be a subclass of __builtin__.str)

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

